packages/feedback-service/src/schema.ts

```diff
@@ -5,18 +5,10 @@ const FeedbackSchema: Schema = new Schema({
   experience: String,
   ticketID: String,
   spa: String,
-  timestamp: {
-    createdAt: { type: Date, default: Date.now },
-    createdBy: {
-      kerberosID: String,
-      name: String,
-    },
-    modifiedAt: Date,
-    modifiedBy: {
-      kerberosID: String,
-      name: String,
-    }
-  },
+  createdAt: { type: Date, default: Date.now },
+  createdBy: { type: String, ref: 'User' },
```

Comment:
This should be just a string, not a ref. Because you do not have access to the User db. So you need to store the user's rhuuid here, and in your resolver, you'll have to fetch the user details from the User microservice (using it's APIs).

_Also a note: Each resolver should send just 1 query to get whatever data you require._